docs/streams/upgrade-guide.html

```diff
@@ -73,8 +73,9 @@ <h1>Upgrade Guide and API Changes</h1>
     </p>
 
     <h3><a id="streams_api_changes_240" href="#streams_api_changes_240">Streams API changes in 2.4.0</a></h3>
-
-    <!-- Placeholder KIP-213 -->
+    <p>
+         As of 2.4.0 Streams now offers a KTable-KTable foreign-key joiner as per (<a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-213+Support+non-key+joining+in+KTable">KIP-213</a>). This joiner allows for records to be joined between two KTables with different keys. Both INNER and LEFT joins are supported.
```

Comment:
nit. Line too long...

`as per (KIP-213)` -> weird `()`? Should it be `(as per KIP-213)` ? Or maybe `(via KIP-213)`